.github/workflows/deploy.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       AUTODEPLOY_TAG: develop
-      AUTODEPLOY_URL: https://dfusion.auto.gnosisdev.com/services/dfusion-v2-api-rinkeby,dfusion-v2-solver-rinkeby,dfusion-v2-api-mainnet,dfusion-v2-solver-mainnet,dfusion-v2-api-xdai,dfusion-v2-solver-xdai/rollout
+      AUTODEPLOY_URL: https://dfusion.auto.gnosisdev.com/services/dfusion-v2-api-rinkeby,dfusion-v2-solver-rinkeby,dfusion-v2-api-mainnet,dfusion-v2-solver-mainnet,dfusion-v2-api-xdai,dfusion-v2-solver-xdai/rollout,dfusion-v2-solver-shadow,dfusion-v2-alerter-mainnet
```

Comment:
I believe this needs to go before `/rollout`

Reply:
Fixed.